mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/MicFragment.kt

```diff
@@ -288,25 +286,6 @@ class MicFragment : MetadataBaseLockFragment(),
         )
     }
 
-    override fun onMediaFilesUploadScheduled() {
```

Comment:
We should leave this code, as we are only migrating to WorkerManager, but we are going to use the same logic to upload the Reports.. (same fore CameraActivity)